src/LaboratoryResults-3-0/Test/XIS-Client/LOINC/TEST-laboratoryresults-LAB-LRS-1-1-send-panelLOINC.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Een algemene opmerking over filenames van de NTS-bestanden:
`TEST-laboratoryresults-LAB-LRS-1-1-send-panelLOINC.xml` - is hier een reden voor/is dit nodig? In de UI van Touchstone ziet dit er niet uit (daar kunnen wij weinig aan doen), dus een deelnemer zit naar tientallen bestanden die beginnen met 'TEST-laboratoryresults-LAB-LRS-' te kijken. Waarom? Als je deelnemers wil helpen het juiste script te vinden, is:
`1-1-send-panelLOINC.xml` dan niet voldoende?